test/sql/window/test_window_exclude.test

```diff
@@ -0,0 +1,558 @@
+# name: test/sql/window/test_window_exclude.test
+# description: Windows with EXCLUDE clause
+# group: [window]
```

Comment:
Very thorough!  The one thing that is missing is testing the other accelerators. So if we could add tests for:

- `COUNT(*)`
- `MEDIAN()`
- Partitioning without ordering ("constant window"). Or can this not happen?

Reply:
Partitioning without ordering is indeed possible. I've added some tests for it.